src/components/sparqlGraph/SparqlGraph.tsx

```diff
@@ -8,8 +8,9 @@ import { ElementDefinition } from 'cytoscape';
 import CytoscapeComponent from 'react-cytoscapejs';
 import { RdfIndividual, RdfSelection, RdfTriple } from '../../models';
 import { rdfObjectKey, rdfPredicateKey, rdfSubjectKey } from './cytoscapeDataKeys';
+import { defaultStyling as defaultUiConfig } from '../../models/uiConfig';
 
-export const SparqlGraph = ({ turtleString, layoutName, onElementsSelected }: SparqlGraphProps) => {
+export const SparqlGraph = ({ turtleString, layoutName, uiConfig, onElementsSelected }: SparqlGraphProps) => {
```

Comment:
```suggestion
export const SparqlGraph = ({ turtleString, layoutName, styles, onElementsSelected }: SparqlGraphProps) => {
```
mb `styles`? :)

Reply:
The object does not only contain styling, but stuff like zoom etc.